autopush/metrics.py

```diff
@@ -62,9 +67,12 @@ def timing(self, name, duration, **kwargs):
 class TwistedMetrics(object):
```

Comment:
Why do these implementations of `IMetrics` not inherit `IMetrics`?

Reply:
good catch.